tests/Microsoft.Identity.Test.Common/TestCommon.cs

```diff
@@ -114,5 +116,18 @@ public static IServiceBundle CreateDefaultAdfsServiceBundle()
             {                
             };
         }
+
+        public static string GetCCSHeader(HttpSnifferClientFactory factory)

```

Comment:
how do you know the header contains the correct values and that clientinfo is returned or not from the /authorize endpoint?
any impact on B2C or ADFS?

There are no tests for `AddCCSHeadersToTokenClient()`, unless you use the ClientInfo class directly.

Reply:
This was handled by a test that created previously. I decided to refactor things a bit to make it more clear though.

Reply:
ADFS and B2C tests are unaffected by the CCS routing info.

Reply:
do they return client info from the /authorize endpoint?

Reply:
is client info returned by B2C and ADFS when going to the /authorize endpoint?

Reply:
It is returned for B2C but not ADFS.

Reply:
so is there any issue w/sending the ccs header to b2c? have we tested to see how b2c handles the additional headers? I assume they are ignored, but would be good to check.

Reply:
Yes, I checked them out. No issues with sending the additional CCS headers when they are not supported. We also have integration tests that send the headers and they are still working. if any issues arise we will know about it.